GeoQuiz/app/src/main/java/com/bignerdranch/android/geoquiz/CheatActivity.kt

```diff
@@ -0,0 +1,75 @@
+package com.bignerdranch.android.geoquiz
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.TextView
+
+private const val TAG = "CheatActivity"
+const val EXTRA_ANSWER_SHOWN = "com.bignerdbranch.android.geoquiz.answer_shown"
+private const val KEY_ANSWER = "answer"
+private const val EXTRA_ANSWER_IS_TRUE =
+    "com.bignerdranch.android.geoquiz.answer_is_true"
+
+class CheatActivity : AppCompatActivity() {
+
+    private lateinit var answerTextView: TextView
+    private lateinit var showAnswerButton: Button
+
+    private var answerIsTrue = false
+    private var isAnswerShown = false
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_cheat)
+
+        answerTextView = findViewById(R.id.answer_text_view)
+        showAnswerButton = findViewById(R.id.show_answer_button)
+        answerIsTrue = intent.getBooleanExtra(EXTRA_ANSWER_IS_TRUE, false)
```

Comment:
it would be easier to identify properties if you would separate views from values when assigning them. please add an empty line above